interface/reports/encounters_report.php

```diff
@@ -53,8 +54,8 @@ function show_doc_total($lastdocname, $doc_encounters) {
   }
 }
 
-$form_from_date = fixDate($_POST['form_from_date'], date($DateFormat));
-$form_to_date = fixDate($_POST['form_to_date'], date($DateFormat));
+$form_from_date = fixDate($_POST['form_from_date'], date('Y-m-d'));
```

Comment:
I was getting a goofy looking date when I was in MM/DD/YYYY format so I switched it back till I could spend some time later looking into it.